packages/graphql-language-service-types/src/index.ts

```diff
@@ -281,23 +281,11 @@ export type CustomValidationRule = (
   context: ValidationContext,
 ) => Record<string, any>;
 
-export type Diagnostic = {
-  range: Range;
-  severity?: number;
-  code?: number | string;
-  source?: string;
-  message: string;
-};
+export type Diagnostic = DiagnosticType;
 
-export type CompletionItem = {
-  label: string;
-  kind?: number;
-  detail?: string;
-  sortText?: string;
-  documentation?: string | null | undefined;
-  // GraphQL Deprecation information
-  isDeprecated?: boolean | null | undefined;
-  deprecationReason?: string | null | undefined;
+export type CompletionItem = CompletionItemType & {
+  isDeprecated?: boolean;
+  deprecationReason?: string;
```

Comment:
Isn't it `string | null | undefined`? Because [deprecationReason: Maybe\<string>;](https://github.com/graphql/graphql-js/blob/b4bff0ba9c15c9d7245dd68556e754c41f263289/tstypes/type/definition.d.ts#L729) in `GraphQLEnumValue` interface and [deprecationReason?: Maybe\<string>;](https://github.com/graphql/graphql-js/blob/b4bff0ba9c15c9d7245dd68556e754c41f263289/tstypes/type/definition.d.ts#L510) in `GraphQLField` interface.

Reply:
good catch! string | undefined will have to do for now. ill fix this in the symbols support PR